### DISCUSSION TITLE
Google Places

### DISCUSSION THREAD

**Discussion by spenserschwartz** [DefinitelyTyped/DefinitelyTyped#69009] [Request a new @types package]
Following up on [Discussion #68943 ](https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/68943) that I initiated, it seems that there is no explicit typing for Google Places API, which has distinct differences from the Google Maps API. See discussion for examples of differences as well as links to the types that I'm spotlighting.

**Comment by Rednegniw** [comment]
I second this, this would help quite a bit.
